waspc/src/Parser/Auth.hs

```diff
@@ -58,12 +65,17 @@ authProperty :: Parser AuthProperty
 authProperty =
   authPropertyUserEntity
     <|> authPropertyMethods
-    <|> authPropertyOnAuthFailedRedirectTo
+    <|> try (authPropertyOnAuthFailedRedirectTo)
```

Comment:
Hey @Martinsos yeah, the try was needed because they both share the same prefix and the match will partially succeed and consume tokens, maybe its better as `(try authPropertyOnAuthFailedRedirectTo <|> authPropertyOnAuthSucceededRedirectTo)`

Reply:
Makes sense, just asking to make sure! Hm, that might be a bit more obvious -> sure let's do that!